src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -56,4 +56,7 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="NHSpecificTest\GH1226\" />
+  </ItemGroup>
```

Comment:
Thanks, but, this change too should not be needed. We have migrated to the new csproj format also for ceasing having changes in it just for adding a test. Is there a specific reason this test would require this change while the other tests do not?

Reply:
I missed this changes in the VS file format, thanks